gtsfm/view_graph_estimator/cycle_consistent_rotation_estimator.py

```diff
@@ -186,14 +198,15 @@ def __save_plots(
         plt.ylabel("Rotation error w.r.t GT")
         plt.axis("equal")
         plt.legend(loc="lower right")
-        plt.savefig(os.path.join("plots", f"gt_err_vs_{self._edge_error_aggregation_criterion}_agg_error.jpg"), dpi=400)
+        plt.savefig(
+            os.path.join(output_dir, f"gt_err_vs_{self._edge_error_aggregation_criterion}_agg_error.jpg"), dpi=400)
```

Comment:
Is this backwards compatible, ie is the same output location used now by default? Isn’t the output not going into a “plots” dir anymore?

Reply:
The new output location is the "debug/cycle_consistency" directory instead of "plots". Do you think I should make it "debug/cycle_consistency/plots"?

Reply:
I suggested moving all intermediate results saved by gtsfm processes to the debug/ folder. I feel like these plots are less useful to end users but more useful to us for debugging.